### PR TITLE
3.0.2 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 3.0.2 (2019-05-17)
+
+- Fixed autocomplete component to emit a `selectionChange` event when input is cleared. [#22](https://github.com/blackbaud/skyux-lookup/issues/22)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: We're back on with articles, yeah? 

Fixed the autocomplete component to emit a `selectionChange` event when the input is cleared.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+- Fixed autocomplete component to no longer have improper Angular form control states when initialized with a value. [#20](https://github.com/blackbaud/skyux-lookup/issues/20)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Fixed the autocomplete component to remove improper Angular form control states when initialized with a value.

Does "remove" change the meaning?

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Hmm... well, it doesn't remove the states. It sets them properly. How about I use the positive tone and shorten it a bit. Something like this... 
`Fixed the autocomplete component to have proper Angular form control states when initialized with a value.`?

Comment (Blackbaud-DeniseCisneros) [reply CHANGELOG.md]: Much better!

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: done.

**Pull request merged**